src/app/js/formats/DefaultFormat/DefaultView.js

```diff
@@ -40,7 +40,7 @@ const DefaultView = ({
 }) => {
     let value = resource[field.name];
     if (Array.isArray(value)) {
-        return <p>{polyglot.t('bad_format', { label: field.label })}</p>;
+        return <p>{polyglot.t('bad_value_format', { label: field.label })}</p>;
```

Comment:
It should be an InvalidFormat or at least an AdminOnlyError or something that is nto displayed to the user here.